src/Announcer.h

```diff
@@ -35,7 +35,8 @@
 #endif
 #include "post_guard.h"
 
-#if defined(Q_OS_LINUX) || defined(Q_OS_OPENBSD)
+#if !defined(Q_OS_MACOS) && !defined(Q_OS_WIN32)
```

Comment:
This logic is getting to be really confusing. Is there a simpler version?

Reply:
Well how would *you* write NOT on MacOs NOR WINDOWS - but everywhere else?

Reply:
Why everywhere else? We only support Linux - and you support BSD's. Add another statement in for whichever one you'd like to see there?

Reply:
Because it is simpler and extensible to exclude the cases where it is **not** required than to *include everything* by default. As it was compilation will fail as soon as you try any "other" OS - whereas that construct will behave - for instance on GNU/Hurd or anything else that someone might through Mudlet into.

This is also consistent now with how the project files are setup - making the "Unix" case (for some values of a POSIXish OS) the default but with modifications for the MacOS and Windozes OSes.